rclcpp/src/rclcpp/clock.cpp

```diff
@@ -15,6 +15,8 @@
 #include "rclcpp/clock.hpp"
 
 #include <memory>
+#include <mutex>
```

Comment:
Nitpick: since the `clock.hpp` includes this header already we usually don't repeat the include in the `cpp` file.

Reply:
Last commit amended to keep history clean.